Submitty/Submitty
# [Bugfix:InstructorUI] Fix duplicate null section on CSV upload

### Please check if the PR fulfills these requirements:


### What is the current / new behavior?
Fixes #7316

(Already described the bug and expected behavior with screenshots attached in the issue, so I won't repeat for conciseness).

### How to test
1. Log in as 'instructor' and create a new blank course for testing
2. go to "Manage Students" from course's sidebar
3. try uploading a classlist CSV file with "NULL" registration section such as [this](https://docs.google.com/spreadsheets/d/1hyubCKktMUGXLyTkgxQkd4cjTengNjEzeQjkhak2VN0/edit?usp=sharing)
4. Check in "Manage Sections" from sidebar now that only one NULL registration section exists in the course. You can also check the actual database more legitimately by doing `SELECT * FROM sections_registration` or `SELECT user_id, registration_section FROM users;` on your test course in pg_admin.
